tools/build.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
-set -e
+set -x -e
```

Comment:
ok. I was wondering whether this had been left over from debugging. perhaps add a comment?